freshmaker/errata.py

```diff
@@ -273,15 +270,11 @@ def get_docker_repo_tags(self, errata_id):
                         "returned None.")
             return None
 
-        repo_tags = dict()
-        for build_nvr in response:
-            if build_nvr not in repo_tags:
-                repo_tags[build_nvr] = dict()
-            repos = response[build_nvr]['docker']['target']['repos']
-            for repo in repos:
-                tags = repos[repo]['tags']
-                repo_tags[build_nvr][repo] = tags
-        return repo_tags
+        return {
+            repo: repo_data['tags']
+            for build_data in response.values()
+            for repo, repo_data in build_data['docker']['target']['repos'].items()
```

Comment:
Using similar change in metaxor as reference: Shouldn't this use key `'external_repos'` instead of `'repos'`?

Reply:
hmm so the original method uses `repos` (https://github.com/redhat-exd-rebuilds/freshmaker/blob/master/freshmaker/errata.py#L283) and when I run unit tests using 
`for repo, repo_data in build_data['docker']['target']['external_repos'].items()` there is a key_error on `external_repos`.

Reply:
Do unit tests have correct data?

Reply:
yup, I believe so because when _response_ is created from passing the _errata_id_ into `xmlrpc.get_advisory_cdn_docker_file_list`, we get a dict that has "repos" as one of the keys. The unit tests reflect this so I believe it's correct. However, in case I am mistaken do you think we should change the 
`xmlrpc.get_advisory_cdn_docker_file_list` response to be 'external_repos' instead of 'repos'? (and make the same change in this method?
![Screenshot from 2021-11-24 00-21-15](https://user-images.githubusercontent.com/44615679/143180745-7fa4d386-7263-4b6b-9337-d0890101ae1e.png)
m

Reply:
The "external_repos" is not mentioned in ET XMLRPC docs but I found it in ET source code. Looks like the "external_repos" value is compatible with quay.io - not sure if we need it here.

Reply:
ahh changes made :+1: